framer/scripts/templates/slider.txt

```diff
@@ -1,21 +0,0 @@
-{{=« »=}}
-import * as React from 'react';
-import { addPropertyControls, ControlType } from 'framer';
-import Mui«componentName» from '@material-ui/core/«componentName»';
-
-interface Props {
-«& tsInterface»
-}
-
-export function «componentName»(props: Props): JSX.Element {
```

Comment:
Does it mean no more slider in framer? 😆

Reply:
Now that I moved the Styled & Unstyled Slider components to the lab I reverted these changes. There were some issues with framer I added TODO in the PR description, but now they are back (until we will move the new components to code and have to deal with it :)